README.md

```diff
@@ -1 +1,21 @@
-# open-auction
\ No newline at end of file
+# OpenAuction
+
+A summary of the project.
```

Comment:
I updated the PR with some changes. Check my last commit and update the summary and "How it Works?" on this file.